Samples/TollApp/VSProjects/TollApp/Program.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using Microsoft.Azure.WebJobs;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+using Newtonsoft.Json;
+using TollApp.Configs;
+using TollApp.Events;
+using TollApp.Models;
+using TollApp.Senders;
+using TollApp.Utils;
+
+namespace TollApp
+{
+    // To learn more about Microsoft Azure WebJobs SDK, please see http://go.microsoft.com/fwlink/?LinkID=320976
+    public class Program
+    {
+        #region Private variables 
+
+        private static Timer _timer;
+        #endregion
+
+        #region Public Methods
+
+        public static void Main(string[] args)
+        {
+            JobHost host = new JobHost();
+            host.Call(typeof(Program).GetMethod("SendData"));
+        }
+
+        [NoAutomaticTrigger]
+        public static void SendData()
+        {
+            var commercialVehicleRegistration = AzureResourcesCreator.CreateBlob();
+            AzureResourcesCreator.CreateAzureCosmosDb();
+            var eventHubSender = new EventHubSender();
+
+            try
+            {
+                // generate data
+                var generator = TollDataGenerator.Generator(commercialVehicleRegistration);
+
+                var timerInterval = TimeSpan.FromSeconds(Convert.ToDouble(CloudConfiguration.TimerInterval));
+
+                TimerCallback timerCallback = state =>
+                {
+                    var startTime = DateTime.UtcNow;
+                    generator.Next(startTime, timerInterval, 5);
+
+                    foreach (var tollEvent in generator.GetEvents(startTime))
```

Comment:
Emitting event once in a while will help

Reply:
Log the event counts to the web job log, to indicate the job is generating events.